session/session.go

```diff
@@ -0,0 +1,144 @@
+package session
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// PortalData ...
+type PortalData struct {
+	AppleID              string              `json:"apple_id"`
+	Password             string              `json:"password"`
+	ConnectionExpiryDate string              `json:"connection_expiry_date"`
+	SessionCookies       map[string][]Cookie `json:"session_cookies"`
+}
+
+// Cookie ...
+type Cookie struct {
+	Name      string `json:"name"`
+	Path      string `json:"path"`
+	Value     string `json:"value"`
+	Domain    string `json:"domain"`
+	Secure    bool   `json:"secure"`
+	Expires   string `json:"expires,omitempty"`
+	MaxAge    int    `json:"max_age,omitempty"`
+	Httponly  bool   `json:"httponly"`
+	ForDomain *bool  `json:"for_domain,omitempty"`
+}
+
+const cookieTemplate = `- !ruby/object:HTTP::Cookie
+  name: <NAME>
+  value: <VALUE>
+  domain: <DOMAIN>
+  for_domain: <FOR_DOMAIN>
+  path: "<PATH>"
+`
+
+// GetSession will fetch the session from Bitrise for the connected Apple developer account
+// If the BITRISE_PORTAL_DATA_JSON is provided (for debug purposes) it will use that instead.
+func GetSession() (string, error) {
+	portalData, err := getDeveloperPortalData(os.Getenv("BITRISE_BUILD_URL"), os.Getenv("BITRISE_BUILD_API_TOKEN"))
```

Comment:
One of a main reason why we try to minimize the occurrence of using type in names:
![Screen Shot 2019-03-19 at 10 35 42 AM](https://user-images.githubusercontent.com/1564933/54595427-e017ff00-4a32-11e9-92fc-c980bcccfedb.png)

Reply:
Fixed in:
https://github.com/bitrise-io/steps-deploy-to-itunesconnect-deliver/pull/61/commits/e254c7963832e39211f47b32764c9d0ff4bd246e